components/menu/metadata/menu.yml

```diff
@@ -1279,10 +1279,10 @@ examples:
                 </div>
               </li>
               <li class="spectrum-Menu-item" role="menuitem" tabindex="0">
+                <svg class="spectrum-Icon spectrum-Icon--sizeM spectrum-Menu-itemIcon spectrum-Menu-itemIcon--workflowIcon" focusable="false" aria-hidden="true" aria-label="Select Add Icon">
+                  <use xlink:href="#spectrum-icon-18-SelectAdd"></use>
+                </svg>
                 <label class="spectrum-Switch-label spectrum-Menu-itemLabel" for="switch-onoff-4">
```

Comment:
Should this change in markup be noted in the migration guide?

Reply:
I've added to the migration guide and update the PR description to reflect breaking change.